src/main/java/com/codecool/terraformingmarsforum/controller/UserController.java

```diff
@@ -1,11 +1,30 @@
 package com.codecool.terraformingmarsforum.controller;
 
 
+import com.codecool.terraformingmarsforum.model.User;
+import com.codecool.terraformingmarsforum.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.net.URI;
+
+
 @RestController
 @RequestMapping("/api/users")
+@RequiredArgsConstructor
 public class UserController {
 
+    private final UserService userService;
+
+    @PostMapping("/")
+    public ResponseEntity<?> createUser(@RequestBody User user) {
```

Comment:
In this case, you send back a 'User' object, so return type should be 'ResponseEntity<User>

Reply:
Done